examples/parsefloat.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+
+#define PARSE_AND_PRINT(type, scan_fmt, print_fmt, str)       \
+  do {                                                        \
+    type val; int ret;                                        \
+    ret = sscanf(str, scan_fmt, &val);                        \
+    printf("\"%s\" => " print_fmt " = %d\n", str, val, ret);  \
+  } while (0)
+
+int main()
+{
+    PARSE_AND_PRINT(float, "%f", "%f", "0.3715");
```

Comment:
Please add test cases to test/libc/stdio/sscanf_test.c

One way you can test floating point numbers is to do what the test/libc/tinymath/ tests do, and round trip it back to a string.

```c
  EXPECT_STREQ(".962423650119207", gc(xdtoa(_acosh(1.5))));
```

Reply:
Will do. I realized there was a test suite minutes after I sent the PR.

Reply:
Added the tests!